src/index.js

```diff
@@ -0,0 +1,177 @@
+    landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (70 <= temp && temp <= 79) {
+    color = 'orange';
+    landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (60 <= temp && temp <= 69) {
+    color = 'brown';
+    landscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (50 <= temp && temp <= 59) {
+    color = 'green';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    color = 'purple';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+
+  const landscapeDisplay = document.getElementById('ground-display');
+  landscapeDisplay.textContent = landscape;
+  const displayTemp = document.getElementById('temp');
+  displayTemp.style.color = color;
+  displayTemp.textContent = String(state.temp);
+};
+
+const resetName = () => {
+  let cityReset = document.getElementById("city-name-input");
+  cityReset.value = "Seattle";
+  changeName();
+
+  // state.city = 'Seattle';
+  // let name = document.getElementById('city-name-display');
+  // name.textContent = state.city;
+  // let newName = document.getElementById('city-name-input');
+  // newName.value = state.city;
+};
+
+const changeName = () => {
+  let name = document.getElementById('city-name-display');
+  let newName = document.getElementById('city-name-input'); //newName = inputName
+  
+  state.city = newName.value;
+  name.textContent = state.city;
+};
+
+const changeSky = () => {
+  const skyDisplay = document.getElementById('sky-display');
+  const skyBackground = document.getElementById('landscape-display');
+  const skyChoice = document.getElementById('sky-choice').value;
+  let skyEmoji;
+  let skyColor;
+
+  // console.log(skyDisplay);
+  // console.log(skyBackground);
+  // console.log(skyChoice);
+
+  if (skyChoice === 'Sunny') {
+    skyEmoji = '☁️ ☁️ ☀️ ☁️ ☁️';
+    skyColor = '#cbedfb';
+  } else if (skyChoice === 'Cloudy') {
+    skyEmoji = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    skyColor = '#b0e0e6';
+  } else if (skyChoice === 'Rainy') {
+    skyEmoji = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    skyColor = '#B0C4DE';
+  } else if (skyChoice === 'Snowy') {
+    skyEmoji = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    skyColor = '#F0F8FF';
+  }
+
+  skyDisplay.textContent = skyEmoji;
+  skyBackground.style.backgroundColor = skyColor;
+};
+
+const registerEvents = () => {
+  tempGroundColorChange();
+
+  const upButton = document.getElementById('temp-up');
+  upButton.addEventListener('click', tempUp);
+  const downButton = document.getElementById('temp-down');
+  downButton.addEventListener('click', tempDown);
+  const getRealTime = document.getElementById('real-time-temp');
+  getRealTime.addEventListener('click', getLongLat);
+
+  changeName();
+  const input = document.getElementById('city-name-input');
+  input.addEventListener('input', changeName);
+
+  const reset = document.getElementById('reset');
+  reset.addEventListener('click', resetName);
+
+  const skyElement = document.getElementById('sky-choice');
+  skyElement.addEventListener('change', changeSky);
+};
+
+document.addEventListener('DOMContentLoaded', registerEvents);
```

Comment:
Nice job registering! Consider using the more robust form or registering that was at the bottom of the crab fan site example.